app/views/users/_customization.html.erb

```diff
@@ -14,7 +14,8 @@
     <fieldset class="crayons-field">
       <legend class="crayons-field__label">Site Theme</legend>
       <div class="theme-selector-field grid gap-4 grid-cols-2 l:grid-cols-3">
-        <% Users::Setting.config_themes.keys.each do |theme| %>
+        <%# This is temporary while we're migrating themes %>
+        <% %w[light_theme dark_theme].each do |theme| %>
```

Comment:
@jeremyf After this PR has been deployed across the fleet this can be changed back (the third PR should take care of it) because the available themes and the themes users can select will be in sync again.